src/display.rs

```diff
@@ -261,17 +267,33 @@ impl Display {
         self.size_info.cell_height = ((metrics.line_height + f64::from(config.font().offset().y)) as f32).floor();
     }
 
+    pub fn replace_glyph_cache(&mut self, config: &Config) -> Result<(), Error> {
+        let (glyph_cache, cell_width, cell_height) =
+            Self::new_glyph_cache(&self.window, &mut self.renderer, config)?;
```

Comment:
Instead of creating a new glyph cache, it should be possible to alter the `update_glyph_cache` method, right? That might be a more efficient way to do things.

Reply:
I tried that initially, but I couldn't get it to work, though I can't now remember why exactly. I also figured since this is generally only going to happen when you move a window between monitors, the efficiency isn't exactly critical. I'm not in a position to do any more work on this for a little while, but I can revisit this later if it's a blocker.

Reply:
I can take a look at it just to make it compile on the weekend, however I'm in no position to actually test it since I only have linux machines.

Generally I think it should be possible to just update the glyph cache since we already do that for font size changes and DPI changes are basically nothing else.

Saying that this "shouldn't be critical" is probably an easy way to get bitten by it in the future. If there's an option to make something faster I think, following the general goal of the project, that option should be taken.

Thanks for all the work though, I'll report back if I managed to get it to work or if I forgot to do it over the weekend. :D

Reply:
Sounds good!

iirc, i got it to compile using `update_glyph_cache`, it just wouldn't do anything when switching monitors. I'm on a break before I start a new job and am without multiple monitors to test this on until I get a new work laptop. I might be able to at least test whatever you have next weekend.

Reply:
It should possible for me to test this in combination with the PR for disabling DPI scaling, I'll test it in there and then add a solution to that PR if I have one.

Then you would be able to rebase on top of that.

Reply:
hey, I think I should be able to work on this, or test something if you have it on a branch.